VikingQuiz.UI/src/Components/Buttons/LoginSignUpButtons/Buttons.css

```diff
@@ -14,4 +14,11 @@
 .switchButton:active {
     transition: All 0.2s linear;
     transform: translateY(4px);
+}
+
+@media screen and (max-width: 300px){
+    .switchButton{
+        top:0px; 
```

Comment:
Please format the code. There should be a space between the column ( ':' ) and the value for the property.